server/datastore/mysql/migrations/tables/20231205165248_AlterSoftwareTitlesAddBrowser.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231205165248, Down_20231205165248)
+}
+
+func Up_20231205165248(tx *sql.Tx) error {
+	_, err := tx.Exec((`DELETE FROM software_titles`)) // delete all software titles, it will be repopulated on the next cron
```

Comment:
curious about this, are you worried about the performance of adding the index with existent data?

Reply:
I was mostly looking for a path where didn't need to worry about how to migrate existing titles with the same name+source but different browsers.

Reply:
okay, I think this is not ideal but in reality since this feature hasn't been deployed the `DELETE` is only to account for dev environments so that's fine 👍

Reply:
Yeah, if we were dealing with other than dev environments, I wouldn't consider it a viable option. Ideally, we will fold all the software-related migrations into one for the actual release like we've discussed, in which case, we won't need to include the `DELETE` at all.